src/Installers/Windows/HostOptions/Product.wxs

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<?include ..\AspNetCoreModule-Setup\IIS-Setup\include.wxi ?>
```

Comment:
What about the arm64 installers requires the changes to this file❔

Reply:
That ICE error about 64bit/32bit folders, this was only x86 before, now we build 64bit for ARM64

Reply:
So we pull in include.wxi to get that preprocessor for program files